docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -215,8 +215,37 @@ Imports
 Template style
 ==============
 
-* In Django template code, put one (and only one) space between the curly
-  brackets and the tag contents.
+* In Django template code: 
+
+  The ``{% extends %}`` should be the first non-comment line
```

Comment:
We should render this as a list
```
* In Django template code, the ``{% extends %}`` should be ...

  ...

* Put one (and only one) space between the curly brackets and the tag contents.

  ....

* Put one (and only one) space between the curly brackets,  the ``%``, and the
  tag contents.

  ...
```
etc.

Reply:
Let’s also put the “In Django template code” in a full sentence before the list, something like:

```
Follow the below rules in Django template code.

* The ``{% extends %}`` should be ...
```